### DISCUSSION TITLE
Multi-database support

### DISCUSSION THREAD

**Discussion by vipas84** [pocketbase/pocketbase#5953] [Q&A]
Dear Gani, is there any chance that we could have multi-database (sqlite) support in PocketBase. De-facto you already have it: one main db and one db for logs... But its important to have an ability to create more than one database, i.e. for each user or for analytics purposes (query copy of production db not an original)... so far as I understand its possible to do by deploying additional instances of PocketBase. Looking forward to reading your thoughts about that. Thanks

**Comment by ganigeorgiev** [comment]
No sorry, I agree with the user that replied to your older discussion in https://github.com/pocketbase/pocketbase/discussions/5138


There are just too many edge cases and will complicate a lot of things (see also another older similar discussion in https://github.com/pocketbase/pocketbase/discussions/180).

**Reply by vipas84** [reply]
Thanks for your answer! You did an amazing job with PocketBase!
